typescript/sdk/src/deploy/protocolDeploymentConfig.ts

```diff
@@ -0,0 +1,41 @@
+import { IsmType } from '../ism/types.js';
+import { ChainTechnicalStack } from '../metadata/chainMetadataTypes.js';
+
+/**
+ * @notice An array of chain technical stacks that are not supported for static deployment.
+ */
+export const skipStaticDeployment: ChainTechnicalStack[] = [
+  ChainTechnicalStack.ZKSync,
+];
+
+export const isIsmStatic: Record<IsmType, boolean> = {
```

Comment:
we already have a "mutable ISM" thing somewhere that is kind of the inverse of this